services/motion/builtin/builtin.go

```diff
@@ -262,3 +275,20 @@ func (ms *builtIn) GetPose(
 		supplementalTransforms,
 	)
 }
+
+func stop(
+	ctx context.Context,
+	resources map[string]referenceframe.InputEnabled,
+	output []map[string][]referenceframe.Input,
+) error {
+	for _, step := range output {
+		for name := range step {
+			if actuator, ok := resources[name].(resource.Actuator); ok {
```

Comment:
all components that are input enabled are also resource.actuators, but not all actuators are input enabled.
Should I modify the interface of input-enabled to encompass resource.actuators?
This was would wouldn't have to cast.

Reply:
I would be okay with this. I'm having trouble imagining something we would make InputEnabled that we would not want to also fulfill the Actuator interface.

Reply:
I like the idea too. This also kinda took me by surprise, I wasn't expecting you to already go ahead and give a shot at implementing my suggestion in bullet 3.

Reply:
I like the idea too. This also kinda took me by surprise, I wasn't expecting you to already go ahead and give a shot at implementing my suggestion in bullet 3.

Let's see what falls out of engineering planning and ticket triage tomorrow before we make any more changes.